src/main/java/Event.java

```diff
@@ -0,0 +1,27 @@
+public class Event extends Task {
+    private String from;
+    private String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    public Event(String input) {
+        super(input.substring(input.indexOf(" ") + 1, input.indexOf("/from") - 1));
+        this.from = input.substring(input.indexOf("/from") + 6, input.indexOf("/to") - 1);
+        this.to = input.substring(input.indexOf("/to") + 4);
```

Comment:
The use of magic numbers is not exactly encouraged in the course. Consider making them as constants? And to go one step further, you might want to consider letting some other part of the program handle input processing/parsing, rather than the Event class itself?